runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -267,10 +270,24 @@ public void onPlayerLootReceived(final PlayerLootReceived playerLootReceived)
 		}
 	}
 
+	@Subscribe
+	public void onScriptCallbackEvent(ScriptCallbackEvent e)
+	{
+		if (!"confirmClanKick".equals(e.getEventName()))
+		{
+			return;
+		}
+
+		final String[] stringStack = client.getStringStack();
+		final int stringSize = client.getStringStackSize();
+		kickPlayerName = stringStack[stringSize - 1];
+	}
+
+
 	@Subscribe
 	public void onChatMessage(ChatMessage event)
 	{
-		if (event.getType() != ChatMessageType.GAMEMESSAGE && event.getType() != ChatMessageType.SPAM && event.getType() != ChatMessageType.TRADE)
+		if (event.getType() != ChatMessageType.GAMEMESSAGE && event.getType() != ChatMessageType.SPAM && event.getType() != ChatMessageType.TRADE && event.getType() != ChatMessageType.FRIENDSCHATNOTIFICATION)
```

Comment:
This is over 200 cols long now... Let's break it onto new lines per condition.
```suggestion
		if (event.getType() != ChatMessageType.GAMEMESSAGE
			&& event.getType() != ChatMessageType.SPAM
			&& event.getType() != ChatMessageType.TRADE
			&& event.getType() != ChatMessageType.FRIENDSCHATNOTIFICATION)
```